components/glew/glew.p5m

```diff
@@ -0,0 +1,62 @@
+#
+# This file and its contents are supplied under the terms of the
+# Common Development and Distribution License ("CDDL"), version 1.0.
+# You may only use this file in accordance with the terms of version
+# 1.0 of the CDDL.
+#
+# A full copy of the text of the CDDL should have accompanied this
+# source.  A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+#
+
+#
+# Copyright 2016 Jim Klimov. All rights reserved.
+#
+
+set name=pkg.fmri value=pkg:/$(COMPONENT_FMRI)@$(IPS_COMPONENT_VERSION),$(BUILD_VERSION)
+set name=pkg.summary value="$(COMPONENT_SUMMARY)"
+set name=info.upstream-url value="$(COMPONENT_PROJECT_URL)"
+set name=info.source-url value="$(COMPONENT_ARCHIVE_URL)"
+set name=info.classification value="$(COMPONENT_CLASSIFICATION)"
+set name=org.opensolaris.consolidation value="$(CONSOLIDATION)"
+
+license $(COMPONENT_LICENSE_FILE) license="$(COMPONENT_LICENSE)"
+
+dir  path=usr
+dir  path=usr/bin
+dir  path=usr/bin/$(MACH64)
+file path=usr/bin/$(MACH64)/glewinfo pkg.depend.bypass-generate=libGL\.so\.1
+file path=usr/bin/$(MACH64)/visualinfo pkg.depend.bypass-generate=libGL\.so\.1
+dir  path=usr/bin/$(MACH32)
+file path=usr/bin/$(MACH32)/glewinfo pkg.depend.bypass-generate=libGL\.so\.1
+file path=usr/bin/$(MACH32)/visualinfo pkg.depend.bypass-generate=libGL\.so\.1
+hardlink path=usr/bin/glewinfo target=../lib/isaexec pkg.linted=true
+hardlink path=usr/bin/visualinfo target=../lib/isaexec pkg.linted=true
+dir  path=usr/include
+file path=usr/include/glew.h
+file path=usr/include/glxew.h
+file path=usr/include/wglew.h
+dir  path=usr/lib
+dir  path=usr/lib/$(MACH64)
+file path=usr/lib/$(MACH64)/libGLEW.a
+file path=usr/lib/$(MACH64)/libGLEW.so.1.13.0 pkg.depend.bypass-generate=libGL\.so\.1
+link path=usr/lib/$(MACH64)/libGLEW.so.1.13 target=libGLEW.so.1.13.0
+link path=usr/lib/$(MACH64)/libGLEW.so target=libGLEW.so.1.13
+file path=usr/lib/$(MACH64)/libGLEWmx.a
```

Comment:
It is a differentiator against earlier releases (that had different licensing) although indeed still not recommended by docs for technical reasons. Fixed earlier.